chronosphere/provider_test.go

```diff
@@ -28,17 +28,41 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// Sanity test that the provider schema can be loaded by TF.
-func TestProviderLoadSchema(t *testing.T) {
-	p := NewProject(t)
+func TestExamples(t *testing.T) {
+	skips := map[string]string{
+		"data-bucket":                "datasources are not supported without a real tenant",
+		"data-collection":            "datasources are not supported without a real tenant",
+		"service-datasource":         "datasources are not supported without a real tenant",
+		"entity-namespaces-internal": "aliased provider configuration not supported by test",
+		"entity-namespaces-managed":  "aliased provider configuration not supported by test",
+		"entity-namespaces-mixed":    "aliased provider configuration not supported by test",
```

Comment:
optional: should we skip these examples, don't think they can ever be useful outside of internal use